SYCL.make

```diff
@@ -93,6 +100,8 @@ SYCLStream.o: SYCLStream.cpp $(SYCL_DEPS)
 SYCLStream.sycl: SYCLStream.cpp
```

Comment:
Wasn't the `.sycl` target only for ComputeCpp anyway, or do the other implementations need it?